test/pylib/manager_client.py

```diff
@@ -173,6 +173,35 @@ async def after_test(self, test_case_name: str, success: bool) -> None:
         logger.info("Cluster after test %s: %s", test_case_name, cluster_status)
 
         return cluster_status
+    
+    async def check_all_errors(self, exclude_errors: list[str] | None = None) -> None:
+        critical_error_pattern = r"Assertion.*failed|AddressSanitizer|Aborting on shard"
```

Comment:
I must say that some of these are full strings that it'd be faster to just search for each without regexp, but that's for a different PR.